Joseph branch merge

Jkwashu opened the pull request:
Styling changes in Report, Graphs added and finished.

Jkwashu committed:
Test for branch

Changed description on Edit.cshtml

Johannes253 committed:
second pie chart

Jkwashu committed:
3rd Pie Chart

Johannes253 committed:
fixed 3rd chart not showing, labels not working

Jkwashu committed:
Fixed Labels on Graph 3

Jkwashu committed:
Graph 4 Working, Label Issues

Jkwashu committed:
Removed item from IntakeDecision

Jkwashu committed:
Fixed labels for Graph 4

Jkwashu committed:
Graph 5 up, Label only showing 0-3

Jkwashu committed:
Graph 6 School Related Arrests Done

Working graph, will need styling later

Jkwashu committed:
Graph 7 Filing Decisions Done Label issues

Labels not showing correctly

Jkwashu committed:
Graph 7 labels fixed, reworked Graph 5 & 6

Jkwashu committed:
Admission graphs finished

Jkwashu committed:
Styling Changes, Rest of Graphs Finished

Johannes253 committed:
fixed savry labels showing up

Johannes253 committed:
doesnt show undefined as header for bar chart anymore

Johannes253 reviewed (commented):
add the text and we are going to talk about everything tomorrow at the meeting. Great job though so far in adding all the charts.

Jkwashu committed:
Styling changes added text to report

Started and added the first half of report text to the right side.

Johannes253 committed:
text actually displaying the correct information

Johannes253 committed:
gender text shows actual values

Johannes253 committed:
age data shows correctly

Johannes253 committed:
intake desciscion numbers showing correctly

Johannes253 committed:
savry text displayed correctly

Jkwashu committed:
Delinquency Graph Code Not working

Jkwashu committed:
Added Database accurate text to right side of graphs

Jkwashu committed:
Added Several Graphs Should be Working

Graphs seem to be working, however there is no current data in the database, leaving an empty array. However this should be grabbing the correct information.

Jkwashu committed:
Delinquency Petition graphs Up, No data to show

Graphs are up and seem to be pulling from correct data, however there is no data to show.

justingrubbs committed:
Filing and Intake decision graphs fixed-ish

justingrubbs reviewed (approved):
Great work!

Johannes253 reviewed (approved):
good job

justingrubbs committed:
Merge branch 'main' into JosephBranch
